projects/eslint-plugin/rules/no-assert-without-ng-dev-mode.js

```diff
@@ -29,7 +29,7 @@ module.exports = {
                         items: {
                             type: 'string',
                         },
-                        description: 'An array of callees that will be checked.',
+                        description: 'An array of callers that will be checked.',
```

Comment:
Cellee is a valid word:
https://en.wiktionary.org/wiki/callee
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/arguments/callee